.azure-pipelines.yml

```diff
@@ -253,22 +253,26 @@ jobs:
       # Check dependency compatibility
       pip check parsec[all]
     displayName: 'Install'
-  # Postgresql tests almost working, but _run function in tests/test_cli.py gives
-  # returncode of 1 instead of 0. Removed test_cli.py from this call as temporary fix
-  # doing tests only on tests/backend/*
-  # - bash: |
-  #     set -eux
-  #     py.test $(pytest.base_args) \
-  #       tests/backend --postgresql --runslow \
-  #       -n auto --max-worker-restart=0 -x \
-  #       --junitxml=test-results-postgresql.xml
-  #   displayName: 'Tests postgresql'
+  - bash: |
+      set -eux
+      py.test $(pytest.base_args) \
+        tests --runmountpoint --runslow \
```

Comment:
`--runmountpoint` enable mountpoint related tests, but this is already tested at line 265... is it expected ?

Reply:
I couldn't figure if there was a difference with the `-m mountpoint` option in line 266. If it's redundant I will remove the mountpoint tests then

Reply:
Nevermind, I get it now